src/app/pages/offline-queue/offline-queue.page.ts

```diff
@@ -24,8 +25,8 @@ export class OfflineQueuePage implements OnInit {
     this.interval = window.setInterval(this.fetchData.bind(this), 2000);
```

Comment:
Interval was hack because we cannot have more than one listener done. I would try to fix multiple listeners observables problem and we can fix this here as well.